orb/docker.go

```diff
@@ -291,8 +298,15 @@ checkContainer:
 		}
 
 		// Bindings
+		bindings := nat.PortMap{
+			"5432/tcp": []nat.PortBinding{{HostPort: "6543", HostIP: "127.0.0.1"}},
```

Comment:
A different port (6543) is selected to avoid conflicts in cases where the user already runs a PostgreSQL server.